com.unity.uiwidgets/Runtime/engine2/UIWidgetsPanel.cs

```diff
@@ -87,7 +87,39 @@ public partial class UIWidgetsPanel : RawImage, IUIWidgetsWindow {
 
         static bool _ShowDebugLog;
 
-        public float devicePixelRatioOverride;
+        public float devicePixelRatioOverride {
```

Comment:
can we put all these to a separate class, since it is not not really a panel specific stuff. assume we have two panels in one scene, they should share this dpr value ?

think carefully on the name, devicePixelRatioOverride is not good (it should be the by-default dpr on devices, instead of overriding something else?)